src/moscot/base/problems/problem.py

```diff
@@ -102,11 +102,11 @@ def _split_mass(arr: ArrayLike) -> ArrayLike:
             data = np.reshape(data, (-1, 1))
         if data.shape[0] != adata.n_obs:
             raise ValueError(f"Expected array of shape `({adata.n_obs,}, ...)`, found `{data.shape}`.")
-        if np.any(data < 0.0):
-            raise ValueError("Some entries have negative mass.")
+        # if np.any(data < 0.0):
+        #    raise ValueError("Some entries have negative mass.")
         total = np.sum(data, axis=0, keepdims=True)
-        if np.any(total <= 0.0):
-            raise ValueError("Some measures have no mass.")
+        # if np.any(total <= 0.0):
+        #    raise ValueError("Some measures have no mass.")
```

Comment:
how are we handling this?

Reply:
Add `# TODO(@MUCDK, @michalk): possibly introduce class property indicating whether we should check for negative mass` and leave it commented out, i.e.leave it as it is and only add this comment :) We will discuss this in the PR to the moscot repo.